content

```diff
@@ -1,7 +1,34 @@
-## Content
-### What is the difference between push, pull, and fetch?
-- `git push` - sent changes from a local branch to a remote repo
```

Comment:
**Suggestion**
Capitalize first word of each description and change to present tense.